Packs/Reco/Integrations/Reco/Reco.py

```diff
@@ -0,0 +1,319 @@
+import base64
+from datetime import datetime
+import dateutil.parser
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+from typing import Any, Dict, Union, Optional, Tuple, List
+
+FILTER_RELATIONSHIP_AND = "FILTER_RELATIONSHIP_AND"
+
+DEMISTO_OCCURRED_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+RECO_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DEMISTO_INFORMATIONAL = 0.5
```

Comment:
1. Rename - informational what?
2. Can we import this constant from their libs?